docs/site/js/customscripts.js

```diff
@@ -64,7 +64,7 @@ $(document).ready(function () {
   var github_requests = [],
   github_stats = JSON.parse(localStorage.getItem('werf_github_stats')) || null;
 
-  function getGithubReuests() {
+  function getGithubRequests() {
```

Comment:
I often include a fix for `Github` => `GitHub`, but didn't look / apply. Happy to include it.